### PR TITLE
Bug fix for get_diary_channel_groups_dataframe

### PR DESCRIPTION
**Pull request by pkaroly**
The function `get_diary_channel_groups_dataframe` returns None if there are no diary_channel_groups to avoid throwing an error.

### CODE REVIEW

File: seerpy/seerpy.py [hunk 1]
```diff
@@ -797,6 +797,9 @@ def get_diary_channel_groups(self, patient_id, from_time, to_time):
     def get_diary_channel_groups_dataframe(self, patient_id, from_time=0, to_time=90000000000000):
         metadata = self.get_diary_channel_groups(patient_id, from_time, to_time)
         channel_groups = json_normalize(metadata).sort_index(axis=1)
+        if len(channel_groups) == 0:
```
Comment (norrishd) [inline seerpy/seerpy.py]: The prescribed way to check if a Pandas DataFrame is empty is just check its attribute: `channel_groups.empty`.

http://pandas-docs.github.io/pandas-docs-travis/reference/api/pandas.DataFrame.empty.html

**Comment by bjdoyle** [comment]
Hey pip do you have your line length set to 80? It should be 100.
It's also better to do a separate format PR rather than reformatting code unrelated to a change.

**Comment by pkaroly** [comment]
MY LINTER WENT MENTAL 🙀  Sorry guys

**Comment by pkaroly** [comment]
> MY LINTER WENT MENTAL 🙀 Sorry guys

I don't know how to fix it :'(

**Pull request closed without merging**